app/Resources/views/page.html.twig

```diff
@@ -256,77 +256,43 @@
-                            selectionFontFamily: 'Georgia, Times, serif',
-                            annotationFontFamily: 'Georgia, Times, serif'
-                        },
-                        openSidebar: '#annotations' === window.location.hash,
-                        disableToolbarCloseBtn: false,
-                        disableToolbarMinimizeBtn: true,
-                        disableToolbarHighlightsBtn: true,
-                        disableToolbarNewNoteBtn: true,
-                        disableBucketBar: true,
-                        enableSidebarDropShadow: true,
-                        enableExperimentalNewNoteButton: true,
-                        enableCleanOnboardingTheme: true,
-                        theme: 'clean',
-                        usernameUrl: '{{ absolute_url(path('profile', {id: 'placeholder'})|replace({'placeholder': ''})) }}',
-                        services: [{
-                            apiUrl: '{{ hypothesis_api }}',
-                            authority: '{{ hypothesis_authority }}',
-                            grantToken: {% if app.user %}'{{ hypothesis_token(app.user) }}'{% else %}null{% endif %},
-                            icon: '{{ absolute_url(asset('assets/favicons/favicon.svg')) }}'
-                            {%- if not app.user %},
-                            onLoginRequest: function () {
-                                window.location.assign('{{ path('log-in') }}');
-                            },
-                            onSignupRequest: function () {
-                                window.location.assign('{{ path('log-in') }}');
-                            }
-                            {%- else %},
-                            onLogoutRequest: function () {
-                                window.location.assign('{{ path('log-out') }}');
-                            },
-                            onProfileRequest: function () {
-                                window.location.assign('{{ path('profile', {id: app.user.username}) }}');
-                            }
-                            {%- endif %}
-                        }],
-                        showHighlights: 'whenSidebarOpen'
-                    };
-                };
-
-                (function () {
-                    var d = document, s = d.createElement('script'), c = d.getElementById('annotation-count');
-                    if (!!c) {
-                        c.dataset.hypothesisAnnotationCount = '';
-                        c.previousElementSibling.dataset.hypothesisTrigger = '';
-                    }
-                    s.src = 'https://hypothes.is/embed.js';
-                    s.async = true;
-                    s.addEventListener('error', function () {
-                        {# Do nothing #}
-                    });
-                    d.head.appendChild(s);
-                })();
-            }
+          if (!!document.createElement('div').dataset) {
```

Comment:
Bit safer to drop this test, and just let the actual JS decide whether to load itself.